lib/auto_html/filters/sanitize.rb

```diff
@@ -1,12 +1,12 @@
-AutoHtml.add_filter(:sanitize).with({}) do |text, options|
-  require 'action_controller'
-  require 'cgi'
-
-  sanitizer_class = if defined?(HTML::WhiteListSanitizer)
-      HTML::WhiteListSanitizer
-    else
-      Rails::Html::WhiteListSanitizer
-    end
+require 'action_controller'
+require 'cgi'
 
-  sanitizer_class.new.sanitize(text, options)
-end
+AutoHtml.add_filter(:sanitize).with({}) do |text, options|
+  if Rails::VERSION::MAJOR == 4 and Rails::VERSION::MINOR >= 2
```

Comment:
I actually prefer to check if `defined?(HTML::WhiteListSanitizer)` than to a fixed version, I think it's more consistent. Can you please remove this change from the PR or split it into two?